src/main/java/graphql/incremental/IncrementalExecutionResultImpl.java

```diff
@@ -91,6 +91,8 @@ public Builder incrementalItemPublisher(Publisher<DelayedIncrementalPartialResul
         public Builder from(IncrementalExecutionResult incrementalExecutionResult) {
             super.from(incrementalExecutionResult);
             this.hasNext = incrementalExecutionResult.hasNext();
+            this.incremental = incrementalExecutionResult.getIncremental();
+            this.incrementalItemPublisher = incrementalExecutionResult.getIncrementalItemPublisher();
```

Comment:
Perhaps you were already in the progress of doing it. Could you please add a test for this?

Reply:
Test please

Reply:
I've added the tests, check out: src/test/groovy/graphql/incremental/IncrementalExecutionResultTest.groovy